src/text-input-autocomplete.directive.ts

```diff
@@ -142,6 +142,7 @@ export class TextInputAutocompleteDirective implements OnDestroy {
   onBlur() {
     if (this.menu) {
       this.menu.lastCaretPosition = this.elm.nativeElement.selectionStart;
+      this.hideMenu();
```

Comment:
Hey, thanks for this!

Let's make this an option and default it to false as it does break existing behaviour: https://travis-ci.org/mattlewis92/angular-text-input-autocomplete/builds/646831695?utm_source=github_status&utm_medium=notification

Something like `@Input() closeMenuOnBlur = false`

and can you also add a test verifying that it works? You should be able to use this one as a starting point: https://github.com/mattlewis92/angular-text-input-autocomplete/blob/master/test/text-input-autocomplete.directive.spec.ts#L184-L198